src/libraries/System.Text.Json/Common/JsonSourceGenerationOptionsAttribute.cs

```diff
@@ -15,11 +15,58 @@ namespace System.Text.Json.Serialization
 #endif
     sealed class JsonSourceGenerationOptionsAttribute : JsonAttribute
```

Comment:
making it sealed prevent users from creation of reusable preconfigured attributes instead of error-prone copy-paste

Reply:
See https://github.com/dotnet/runtime/issues/56206